apps/showcase/src/app/routes/components/routes/file-uploader/file-uploader.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="author"><strong>Author:</strong> Jerrol Krause</div>
+<h1>File Uploader</h1>
+<p>
+  This file uploader component allows users to easily manage file uploads. You will still need to perform the upload to
+  your API after a user has selected their files. It features a intuitive and user-friendly design, with the following
+  capabilities:
+</p>
+<ul>
+  <li>Support for multiple file selection</li>
+  <li>Drag and drop functionality for easy file selection</li>
+  <li>Real-time file validation, checking for file size and type restrictions</li>
+  <li>File previews, including image previews for image files</li>
+  <li>The ability to show a preview of the uploaded files and metadata such as file name and file size</li>
+  <li>The ability to show the corresponding icon for the file type using the file extension</li>
+  <li>Error handling for upload restrictions</li>
+  <li>The ability to remove a single file out of multiple selected</li>
+</ul>
+<hr />
+
+<p-tabView>
+  <!-- Documentation -->
+  <p-tabPanel header="Documentation">
+    <h2>Import</h2>
+    <pre><code class="language-ts" [innerHtml]="import"></code></pre>
+    <h2>HTML</h2>
+    <pre><code class="language-html" [innerHtml]="exampleHTML"></code></pre>
```

Comment:
For my curiosity,
Why simple angular interpolation doesn't work here instead of innerHtml?
Or if it should be treated as html (not text), why you didn't use ng-template?

Reply:
For library components I almost always use innerHTML because I want parent components to have the ability to insert HTML into their inputs. For instance a common task is to insert an icon before some text or wrap some text in a custom class. You can't do that with interpolation. innerHTML is universally more flexible than interpolation and even if the use case today doesn't require HTML it may tomorrow.